src/components/Popups/PopupDrawingToolbar.tsx

```diff
@@ -50,16 +53,24 @@ const PopupDrawingToolbar = ({
     canRedo,
     canUndo,
     className,
+    drawnPathGroups,
     onDelete,
     onRedo,
     onReply,
     onUndo,
     reference,
 }: Props): JSX.Element => {
     const intl = useIntl();
+    const popupRef = React.useRef<PopupBase>(null);
+
+    React.useEffect(() => {
+        const { current: popup } = popupRef;
+        popup.popper.update();
```

Comment:
Is `popup` always defined here? Seems like TS should be throwing an exception here.

Reply:
I added an if case.